contracts/duckies/nft/RandomUpgradeable.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.18;
+
+import '@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol';
+
+contract RandomUpgradeable is Initializable {
```

Comment:
Not clear if it's Proxy upgradable

Reply:
No, the contract itself is not upgradeable. It is made to be used by upgradeable contracts like a library, but it holds some storage values, so it must be declared as a contract.